modules/multicast/multicast.c

```diff
@@ -705,12 +724,41 @@ static const struct cmd cmdv[] = {
 };
 
 
+static void event_handler(enum ua_event ev, struct bevent *event, void *arg)
+{
+	(void)arg;
+	const struct sip_msg *msg  = bevent_get_msg(event);
+
+	switch (ev) {
+
+	case UA_EVENT_SIPSESS_CONN:
+		if (mc.dnd) {
+			(void)sip_treply(NULL, uag_sip(), msg, 480,
+					 "Temporarily Unavailable");
+			info("menu: incoming call from %r <%r> rejected: "
+			     "480 Temporarily Unavailable\n",
+			     &msg->from.dname, &msg->from.auri);
+			bevent_stop(event);
```

Comment:
TODO for baresip repo: use `bevent_stop()` also in baresip

Reply:
done: https://github.com/baresip/baresip/pull/3148

Reply:
baresip-apps\modules\multicast\receiver.c  uag_set_dnd()